domain/model/model.go

```diff
@@ -0,0 +1,7 @@
+package model
+
+type Pokemon struct {
```

Comment:
As a good practice please put a description on structures, functions and interfaces :D